src/utils/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export const getArticles = () => {
+  return axios
+    .get("https://hannahtm-nc-news-backend-project.onrender.com/api/articles")
+    .then((response) => {
+      return response;
```

Comment:
Now that you know your data is always returned on keys of data and then keys, you might want to just `return response.data.articles` here, rather than having to drill into the keys in your App.jsx file on line 22 when setting state